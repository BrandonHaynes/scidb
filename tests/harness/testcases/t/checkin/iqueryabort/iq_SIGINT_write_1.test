--setup
--test

--shell --store --command "${TEST_UTILS_DIR}/killquery.sh -afl 2 2 'store (build (<v : int64> [I=0:2000,1,0], I), ABRTEST)' 1> /tmp/${HPID}_1.stdout || echo FAILURE"
# log the output
--shell --command "cat /tmp/${HPID}_1.stdout"

# this should fail due to the above abort
--error --long=20 "remove(ABRTEST)"

--cleanup

--shell --command "rm -f /tmp/${HPID}_1.stdout 2>/dev/null"
