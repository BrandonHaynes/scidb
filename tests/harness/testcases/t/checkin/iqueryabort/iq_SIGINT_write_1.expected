
[An error expected at this place for the query "remove(ABRTEST)". And it failed with error code = scidb::SCIDB_SE_QPROC::SCIDB_LE_ARRAY_DOESNT_EXIST long code = 20. Expected long code = 20.]

